examples/stack/link_layer/example_link_layer_md.py

```diff
@@ -0,0 +1,100 @@
+import os
+from typing import Any, Dict, Generator
+
+from pydynaa import EventExpression
+from squidasm.run.stack.config import StackNetworkConfig
+from squidasm.run.stack.run import run
+from squidasm.sim.stack.common import LogManager
+from squidasm.sim.stack.program import Program, ProgramContext, ProgramMeta
+
+
+class ClientProgram(Program):
+    PEER = "server"
+
+    def __init__(self, basis: str, num_pairs: int):
+        self._basis = basis
+        self._num_pairs = num_pairs
+
+    @property
+    def meta(self) -> ProgramMeta:
+        return ProgramMeta(
+            name="client_program",
+            parameters={"basis": self._basis, "num_pairs": self._num_pairs},
+            csockets=[self.PEER],
+            epr_sockets=[self.PEER],
+            max_qubits=1,
+        )
+
+    def run(
+        self, context: ProgramContext
+    ) -> Generator[EventExpression, None, Dict[str, Any]]:
+        conn = context.connection
+        epr_socket = context.epr_sockets[self.PEER]
+
+        results = epr_socket.create_measure(number=self._num_pairs)
+
+        yield from conn.flush()
+
+        outcomes = [int(r.measurement_outcome) for r in results]
+
+        return outcomes
+
+
+class ServerProgram(Program):
+    PEER = "client"
+
+    def __init__(self, basis: str, num_pairs: int):
+        self._basis = basis
+        self._num_pairs = num_pairs
+
+    @property
+    def meta(self) -> ProgramMeta:
+        return ProgramMeta(
+            name="server_program",
+            parameters={"basis": self._basis, "num_pairs": self._num_pairs},
+            csockets=[self.PEER],
+            epr_sockets=[self.PEER],
+            max_qubits=1,
+        )
+
+    def run(
+        self, context: ProgramContext
+    ) -> Generator[EventExpression, None, Dict[str, Any]]:
+        conn = context.connection
+        epr_socket = context.epr_sockets[self.PEER]
+
+        results = epr_socket.recv_measure(number=self._num_pairs)
+
+        yield from conn.flush()
+
+        outcomes = [int(r.measurement_outcome) for r in results]
+
+        return outcomes
+
+
+if __name__ == "__main__":
+    # LogManager.set_log_level("WARNING")
```

Comment:
This is a commented line which may also be removed.

Reply:
Removed

Reply:
It still shows up in the changes I see, so is it removed?